Makefile

```diff
@@ -9,7 +9,7 @@ help: ## display a help message
 shell: ## run a shell on the studio-frontend container
 	docker exec -it dahlia.studio-frontend /bin/bash
 
-attach:
+attach: ## attach local standard input, output, and error streams to studio-frontend container
```

Comment:
These commands weren't showing up in the output of `make help`, so I thought it might be because they were missing descriptions, but they still don't show up ;(.

Reply:
Are you sure? When I check out your branch, they appear in the help output:

```
ericfischer@Eric-Fischers-MacBook-Pro:~/tmp_code/studio-frontend$ git fetch; gco thallada/refactor-status-alert
remote: Counting objects: 128, done.
remote: Compressing objects: 100% (37/37), done.
remote: Total 128 (delta 83), reused 125 (delta 83), pack-reused 2
Receiving objects: 100% (128/128), 27.87 KiB | 0 bytes/s, done.
Resolving deltas: 100% (83/83), completed with 25 local objects.
From https://github.com/edx/studio-frontend
 + 1f36134...598444e mroytman/hide-image-previews   -> origin/mroytman/hide-image-previews  (forced update)
 + ea0fbd2...7dc5696 thallada/refactor-status-alert -> origin/thallada/refactor-status-alert  (forced update)
Branch thallada/refactor-status-alert set up to track remote branch thallada/refactor-status-alert from origin by rebasing.
Switched to a new branch 'thallada/refactor-status-alert'
ericfischer@Eric-Fischers-MacBook-Pro:~/tmp_code/studio-frontend$ make help
Makefile
Please use `make <target>' where <target> is one of
  asset-page-flag           insert a waffle flag into local docker devstack
  attach                    attach local standard input, output, and error streams to studio-frontend container
  devstack.update           use this if you don't want to fire up the dedicated asset watching containers
  down                      stop and remove studio-frontend container
  from-scratch              start development environment from scratch
  help                      display a help message
  logs                      show logs for studio-frontend container
  npm-install-%             install specified % npm package on the studio-frontend container
  pull_translations         must be exactly this name for edx tooling support, see ecommerce-scripts/transifex/pull.py
  restart-detached          bring container down and back up in detached mode
  restart                   bring container down and back up
  shell                     run a shell on the studio-frontend container
  up-detached               bring up studio-frontend container in detached mode
  up                        bring up studio-frontend container
```

As far as I can tell, the perl regex in `help`'s definition is explicitly checking for the `##` divider to use as a description in the output

Reply:
That's odd. This is what I see.

```
mroytman:~/workspaces/studio-frontend $ make help
Makefile
Please use `make <target>' where <target> is one of
  asset-page-flag           insert a waffle flag into local docker devstack
  devstack.update           use this if you don't want to fire up the dedicated asset watching containers
  down                      stop and remove studio-frontend container
  from-scratch              start development environment from scratch
  help                      display a help message
  logs                      show logs for studio-frontend container
  npm-install-%             install specified % npm package on the studio-frontend container
  pull_translations         must be exactly this name for edx tooling support, see ecommerce-scripts/transifex/pull.py
  shell                     run a shell on the studio-frontend container
  up-detached               bring up studio-frontend container in detached mode
  up                        bring up studio-frontend container
```